src/common/conditional_compilation/include/openvino/cc/pass/itt.hpp

```diff
@@ -21,6 +21,13 @@ OV_CC_DOMAINS(ov_pass);
 #    define MATCHER_SCOPE(region)         const std::string matcher_name(OV_PP_TOSTRING(region))
 #    define RUN_ON_MODEL_SCOPE(region)    OV_SCOPE(ov_pass, OV_PP_CAT(region, _run_on_model))
 
+#    define ADD_MATCHER_FOR_THIS(nspace, region, ...)        add_matcher<nspace::region>(__VA_ARGS__);
+#    define ADD_MATCHER_FOR_THIS_WITHOUT_NSPACE(region, ...) add_matcher<region>(__VA_ARGS__);
+#    define ADD_MATCHER(obj, nspace, region, ...)            obj->add_matcher<nspace::region>(__VA_ARGS__);
+#    define ADD_MATCHER_WITHOUT_NSPACE(obj, region, ...)     obj->add_matcher<region>(__VA_ARGS__);
+#    define REGISTER_PASS(obj, nspace, region, flag, ...)    obj.register_pass<nspace::region>(__VA_ARGS__);
+#    define REGISTER_DISABLED_PASS(obj, nspace, region, ...) obj.register_pass<nspace::region, false>(__VA_ARGS__);
+#    define REGISTER_PASS_MODEL_IF(region)
```

Comment:
Honestly this macro won't work.

In all place you use it as:
```
REGISTER_PASS_MODEL_IF(GraphRewrite) { ... }
```
In means that this macro will work if ANY GraphRewrite has been called. We need to have unique id in order to enable only necessary parts.

Reply:
Maybe we need extend GraphRewrite class to support unique identificator

Reply:
You mean "GraphRewrite" maybe not unique, right? So maybe we can remove "REGISTER_PASS_MODEL_IF" temporarily.

Reply:
Yes `GraphRewrite` is a common class

Reply:
Updated.